Week1/homework/world-queries.txt

```diff
@@ -1,40 +1,362 @@
-Answers for the world queries go here:
+-- MySQL Workbench Forward Engineering
 
-For example:
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
 
-1. What are the names of all the cities in the Netherlands?
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
 
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Users`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Users` (
+  `Id_User` INT NOT NULL AUTO_INCREMENT,
+  `FirstName_User` VARCHAR(45) NULL,
+  `LastName_User` VARCHAR(45) NULL,
+  `Email_User` VARCHAR(45) NULL,
```

Comment:
There is no need to add the table name to all the column names. The more common convention is not to use the table name.